microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,83 @@
+title: Feature differences between Microsoft Edge and WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/08/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, WebView2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Feature differences between Microsoft Edge and WebView2  
+
+WebView2 is based off of the Microsoft Edge browser.  You \(a WebView2 developer\) have the opportunity to extend features from the browser to WebView2-based apps. However, WebView2 is not limited to only being a browser. There are many scenarioin which the browser's features are unneccesary or intrusive.     
+
+The purpose of this article is to provide the following information.  
+
+*   The current approach that the WebView2 team uses for browser features.  
+*   The status of features and the reasoning behind the state.  
+*   The ability for features to be turned on \(or turned off\).  
+
+## Guiding Principles  
+
+Browser features within the context of WebView2 should adheres to the following guidelines for most WebView2 features.  
+
+*   Most of the WebView2 features work the same as Microsoft Edge features.  The following exceptions define why a feature is turned off.  
+    *   It doesn't make sense within the context of WebView2.  
+    *   It has an extended service cost.  
+*   The WebView2 features shouldn't include any Microsoft Edge branding.  
+    
+## Features  
+
+The following list displays the WebView2 features that differ from the new Microsoft Edge browser.  
+
+| Feature | Default | Configurable | Details |  
+|:--- |:--- |:--- | :--- | 
+| Autofill for Addresses | Enabled | Yes | This feature is turned on by default, you may turn it on or off. |  
+| Autofill for Passwords | Enabled | Yes | This feature is turned on by default, you may turn it on or off. |  
+| Autofill for Payments | Disabled | No | This feature is turned off. | 
+| Browser Extensions | Disabled | No | This feature is turned off. |   
+| Browser Task Manager | Disabled | No | This feature is turned off. |  
+| Continue-where-I-left-off prompt | Disabled | No | This feature is turned off. |  
+| Downloads | Enabled | Yes | This feature allows you to customize the download UI to manipulate downloads, such as block, redirect, save, pause, and so on.  For more information, navigate to <!--[download api][Webview2ReferenceDownloadApi]-->. |  
+| Family Safety | Disabled | No | This feature is turned off. |  
+| Hotkeys | Disabled | No | The following shortcuts aren't turned off:  `Ctrl`+`F` and `Ctrl`+`S` |  
+| IE Mode | Disabled | No | IE Mode is disabled. |  
+| Immersive Reader | Disabled | No | This feature is dependent on the browser UI for control and triggering.  This feature is turned off. |  
+| Intrusive Ads | Disabled | No | The UI for this feature is removed.  This feature is turned on. |  
```

Comment:
Intrusive Ads and Read Aloud say they are `Disabled` in the enabled column and in details `This feature is turned on`. You probably don't need to repeat if the feature is on or off in details if you already say it in the enabled column.

Reply:
Got it, need to reword this so its not misleading, I think what I was trying to say is the ui for the feature is removed, however the underlying functionality is still enabled in webview2.

Reply:
changed to 'The UI for this feature is removed, however the underlying functionality persists.'